### ISSUE TITLE
Evaluation ONLY mode in MAPE environment

### ISSUE THREAD

**Issue by ConstantinosM** [marlbenchmark/on-policy#16]
From what I understand, in MAPE, evaluation is "entangled" in the training mode (there is alteration between training and evaluation phase). Is there any way (or a script) that I can evaluate only a trained agent and save gifs/videos etc?

**Comment by zoeyuchao** [comment]
Of course u can do that, it is super easy because the training and evaluation code are decoupled.

**Comment by ConstantinosM** [comment]
Thanks a lot, I eventually figured it out by using the render_mpe.py. A couple of issues with this script:

1. There is no gif_dir defined (used at the bottom of the file when imageio saves the gif files) anywhere in the file. I specified my own folder.
2. The way it creates the folder for saving the log files (e.g. for Tensorboard) seems a bit problematic to me (except if I am missing its logic). The script creates a NEW <name (e.g. run)> folder and stores there the log files and potentially the gifs. It might make sense to store these files in the folder where the restored model is in.
